plugins/CoreHome/vue/src/Comparisons/Comparisons.store.ts

```diff
@@ -293,10 +293,13 @@ export default class ComparisonsStore {
   private loadComparisonsDisabledFor() {
     const matomoModule: string = MatomoUrl.parsed.value.module as string;
 
-    if (typeof matomoModule === 'undefined') {
+    // check if body id #installation exist
+    const install = document.getElementById('installation');
```

Comment:
Guess this should be good for now, but it actually might have the risk, that this regresses if any element has this id, maybe from a custom plugin. We could harden the check for `body#installation`, as the body element shouldn't have that id outside installation.